### PR TITLE
Update roadmap with Q3 objectives

### PR DESCRIPTION
**Pull request by stevenjmesser**
There's a few changes in this PR.

Instead of referencing which quarter we're in, I've put 'Things we're working on'. I've also added a 'Things we're exploring' section to differentiate that work from feature work. I've also added an 'Up next' section, to show what we're likely to work on after meeting our main objectives.

I've made these changes because I'm hoping we can update this more regularly. A roadmap is a living document.

### CODE REVIEW

File: source/roadmap.html.erb [hunk 1]
```diff
@@ -59,49 +59,52 @@ title: Roadmap
...
+                <li>Hide sensitive card details when call centre agents are working from home (a <a class="govuk-link" href="https://www.payments.service.gov.uk/take-payments-by-phone-or-post/">MOTO payments</a> feature)</li>
+                <li>Increased the <a class="govuk-link" href="https://docs.payments.service.gov.uk/custom_metadata/#add-custom-metadata">metadata</a> value limit from 50 to 100 characters</li>
+                <li>Reports on bank payouts from Stripe available via CSV and API</li>
+                <li>Performance improvements to the Transactions page in the <a class="govuk-link" href="https://selfservice.payments.service.gov.uk/login">admin tool</a></li>
+                <li>Various operational improvements and bug fixes</li>
+
               </ul>
 
-             <h2 class="govuk-heading-m">July to September 2020</h2>
+             <h2 class="govuk-heading-m">Things we're working on</h2>
```
Comment (katstevens) [inline source/roadmap.html.erb:71]: Do we stick to curly apostrophes as a rule?

Comment (stevenjmesser) [reply source/roadmap.html.erb:71]: Oh, @alexbishop1 usually tidies up my use of the wrong apostrophe! But yes, I think we do use `’` not `'`.

**Review by katstevens — APPROVED** [review]
Cool - apart from that one apostrophe, it looks good to me 👍

**Comment by stevenjmesser** [comment]
Thanks @katstevens, I've added another commit. Could you review again pretty please? Ta!

**Review by katstevens — APPROVED** [review]
I can indeed!

**Comment by stevenjmesser** [comment]
Thank you! Someone else needs to merge it, I don't have the right permissions. Thank you, Merger!

**Pull request merged**